packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -114,7 +114,7 @@ describe(`background sync queue`, function() {
       await backgroundSyncQueue.replayRequests();
       throw new Error('Replay should have failed because of invalid URL');
     } catch (err) {
-      expect(err[0].status).to.equal(404);
+      // TODO: Check for specific error.
```

Comment:
This won't actually test anything - you throw inside the try block meaning: replayRequest() doesn't throw, you then throw 'Replay should have failed...' meaning the catch will be caught and the test will pass.

Changing to:

```
return backgroundSyncQueue.replayRequests()
.then(() => {
    // Expected test to throw an error but it didn't
    throw new Error('...');
}, (err) => {
    // The function throw an error as expected - nothing to do.
})
```

will fix this

Reply:
fixed :)